changelogs/fragments/2246-terraform-validate.yaml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
+  - terraform - fixed issue that cause the execution fail because in Terraform 0.15 -var and -var-file options are no longer available on terraform validate (https://github.com/ansible-collections/community.general/pull/2246). 
```

Comment:
```suggestion
  - terraform - fix issue that cause the execution fail because from Terraform 0.15 ``-var`` and ``-var-file`` options are no longer available on terraform validate (https://github.com/ansible-collections/community.general/pull/2246). 
```